### PR TITLE
policy: Add experimental -mempoolreplacement=full (off by default)

### PR DESCRIPTION
**Pull request by maflcko**
In experimental settings it could make sense to give users the option to enable full replace by fee. The option is off by default, because it could hurt block or transaction propagation for the node that has it enabled.

### CODE REVIEW

**Comment by elichai** [comment]
Sounds like a good idea. it should be the users decision to turn on this feature

File: test/functional/feature_rbf.py [hunk 1]
```diff
@@ -391,7 +408,8 @@ def test_too_many_replacements(self):
             tx_i.vin = [CTxIn(COutPoint(txid, i), nSequence=0)]
             tx_i.vout = [CTxOut(split_value - fee, CScript([b'a' * 35]))]
             tx_i_hex = txToHex(tx_i)
-            self.nodes[0].sendrawtransaction(tx_i_hex, 0)
+            for i in self.RBF_NODES:
```
Comment (practicalswift) [inline test/functional/feature_rbf.py]: Is the redefining on `i` from the outer scope intentional here?

**Comment by TheBlueMatt** [comment]
I'm -0.5 on this one. I'm not sure its really worth much, and I'm unclear on exactly what it solves. That said, #10823 probably solves whatever this solves without needlessly irritating random trolls, and we don't have to make it opt-in.

**Comment by luke-jr** [comment]
See #7219

**Pull request closed without merging**